ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -565,6 +567,8 @@ Install Model Serving Stack Dependencies
     ...                This is likely going to change in the future and it will include a way to skip installation.
     ...                Caikit runtime will be shipped Out-of-the-box and will be removed from here.
     RHOSi Setup
+    ${enabled}=    Is Component Enabled    kserve
```

Comment:
Could you create a keyword for this in ods_ci/tests/Resources/Common.robot?

```
Skip If Component Is Not Enabled
    [Documentation]    Skips test if ${component_name} is not enabled in DataScienceCluster
    [Arguments]    ${component_name}
```

Reply:
done